app/workers/new_page_worker.rb

```diff
@@ -0,0 +1,29 @@
+class NewPageWorker
+  include Sidekiq::Worker
+
+  def perform(page_id)
+    page = Page.find(page_id)
+    make_screenshot(page)
+    parse_html_content(page)
+    puts "Page #{page.id} created!"
+  end
+
+  private
+
+  def make_screenshot(page)
+    image_path_to_save = Rails.root.join('tmp', "page_shot_#{page.id}.png")
+    image = Gastly.capture(page.source_url, image_path_to_save)
```

Comment:
what if URL is not responding? e.g. site is not working.
Add some error handling. 
Basically pass not existing URL and see how it works